application/backend/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>pl.ds.luna</groupId>
     <artifactId>luna-application</artifactId>
-    <version>1.1.1-SNAPSHOT</version>
+    <version>1.2.0-SNAPSHOT</version>
```

Comment:
for the breaking change major version should be bumped

Reply:
Changed to 2.0.0